cabot/metricsapp/models/base.py

```diff
@@ -53,6 +54,16 @@ class Meta:
         default=30,
         help_text='Time range in minutes the check gathers data for.',
     )
+    grafana_panel = models.ForeignKey(
+        'GrafanaPanel',
+        null=True
+    )
+    auto_sync = models.NullBooleanField(
+        default=False,
```

Comment:
I had the default as False for non-Grafana checks--I had auto_sync as a hidden field for them so it'd always be False. 

Changing this to True and filtering for only Grafana checks in sync_all_grafana_checks would also work though